source/index.html.md

```diff
@@ -132,14 +132,257 @@ curl "https://www.castupload.com/api/v1/actor_profiles/123" \
+      },
+      "orientation": "landscape",
+      "dimensions": [
+        365,
+        360
+      ],
+      "main_picture": false,
+      "year": 2021,
+      "recorded_at": null
+    }
+  ],
+  "external_showreel": "https://www.youtube.com/watch?v=xxxx",
+  "external_showreels": [
+    "https://www.youtube.com/watch?v=xxxx",
+    "https://vimeo.com/xxxx"
+  ],
+  "showreels": [
+    {
+      "name": "Showreel 2021",
+      "url": "https://www.castupload.com/actors/john-doe/showreels/15",
+      "type": "video"
+    },
+    {
+      "name": "Showreel 2020",
+      "url": "https://www.castupload.com/actors/john-doe/showreels/16",
+      "type": "video"
+    }
+  ],
+  "vita": {
+    "primary_education": [
+      {
+        "name": "ABC School",
+        "year_from": 2012,
+        "id": "Z2lkOi8vY2FzdHVwbG9hZC9BY3RpbmdTY2hvb2w6OkNvbm5lY3Rpb24vMg"
+      }
+    ],
+    "award": [
+      {
+        "id": 327,
+        "type_genre": "cinema_film",
+        "info": "",
+        "name": "Avanca Film Festival - Best Performance",
+        "year_from": 2017,
+        "year_to": 2017,
+        "award_nominated": true
+      }
+    ],
+    "education": [
+      {
+        "id": 329,
+        "type_genre": null,
+        "info": "",
+        "name": "123 School",
+        "year_from": 2013,
+        "year_to": 2014
+      }
+    ],
+    "film": [
+      {
+        "id": 328,
+        "type_genre": "cinema_film",
+        "info": "",
+        "name": "Schneeflöckchen",
+        "year_from": 2019,
+        "year_to": 2020,
+        "role": "Jack",
+        "role_type": "supporting",
+        "distributor": "",
+        "director": "Adolfo Kolmerer",
+        "producer": "",
+        "caster": ""
+      },
+      {
+        "id": 326,
+        "type_genre": "short_film",
+        "info": "",
+        "name": "The Harmonica ",
+        "year_from": 2018,
+        "year_to": 2018,
+        "role": "Felix",
+        "role_type": "leading",
+        "distributor": "",
+        "director": "",
+        "producer": "Deutsche Film- und Fernsehakademie Berlin",
+        "caster": ""
+      }
+    ],
+    "television": [
+      {
+        "id": 330,
+        "type_genre": "series",
+        "info": "",
+        "name": "Mord mit Aussicht",
+        "year_from": 2015,
+        "year_to": 2016,
+        "role": "Robert",
+        "role_type": "misc",
+        "distributor": "",
+        "director": "",
+        "producer": "",
+        "caster": ""
+      }
+    ],
+    "theatre": [],
+    "audio": [],
+    "other": [],
+    "internet": [],
+    "commercial": []
+  },
+  "claimed": true,
+  "selected_categorizations": [
+    "film_actor"
+  ],
+  "locale": "de",
+  "email": null,
+  "phone": null,
+  "role_types": [
+    "film",
+    "serials",
+    "theatre"
+  ],
+  "available_as_scene_partner": false,
+  "cached_complete": false,
```

Comment:
this example should not contain any internal (i.e. admin) specific attributes, please remove all of them.